### PR TITLE
Readme file  edited 

### PR DESCRIPTION
**Pull request by mayaalsamarrai**
Added details to Functional possibilities chapter: points 1,3,11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,321 @@
...
+	     1. Selectable list of permissions
+
+![Fig.Customer Orders](/docs/media/screen-permission-name.png)
+
+![Fig.Customer Orders](/docs/media/screen-assign-permissions.png)
+
+1. The admin fills out the fields, assigns permissions to the new role and clicks the ‘Create’ button
+1. The new role with permissions is created
+
+**Important**: The admin can select different set of permissions for multiple roles related to Orders Module. Roles and permissions can be both edited and deleted from the system.
```
Comment (yecli) [inline README.md]: Permissions can not be deleted from the system through admin UI, they could be removed from specific role.

File: README.md [hunk 2]
```diff
@@ -1,34 +1,321 @@
...
+
+#### Invoice for Customer Order
+
+This template is used for customer invoice generation in PDF format
+
+![Fig.Customer Orders](/docs/media/screen-invoice-for-customer-order.png)
+
+## Database Model
+
+![Fig.db-model](/docs/media/diagram-db-model.png)
```
Comment (yecli) [inline README.md]: Is it supposed to be model screenshot? I saw class diagram picture [here](https://github.com/VirtoCommerce/vc-module-order/pull/101), but it is named "docs/media/entity-data-model.png".

File: README.md [hunk 3]
```diff
@@ -1,34 +1,321 @@
...
+
+    1. Shipping price
+
+    1. Discount total
+
+    1. Tax total
+
+    1. Total
+
+## Roles and Permissions
```
Comment (yecli) [inline README.md]: This section contains general info on permissions. I think it should be placed in vc-platform repository and linked here.
What's more important to cover in module docs - order module permission details: 
image

File: README.md [hunk 4]
```diff
@@ -1,34 +1,321 @@
...
 
-![Order UI](https://cloud.githubusercontent.com/assets/5801549/15569048/854c646c-232f-11e6-9819-7230045cad0c.png)
+## Overview
 
-# Documentation
-User guide: <a href="https://virtocommerce.com/docs/vc2userguide/order-management" target="_blank">Order Management</a>
+The Orders Module in Virto Commerce is a document based flexible orders management system with possibility to add unlimited number of documents related to customer order.
 
-Developer guide:
+The Orders Module main purpose is to store order details and manage orders created by users on client side. This module is not designed to be a full order processing system like EPR but serves as storage for customer orders details and can be synchronized with different external processing systems.
```
[the remark this answers is not in the document]
Comment (mayaalsamarrai) [reply README.md]: Yes, ERP!

[the remark this answers is not in the document]
Comment (mayaalsamarrai) [reply README.md]: Screenshots quality is not good enough, I cannot see exact labels even after opening pictures and zoom.- agree!!! But I have no idea how to improve the quality using the light shot. May be you can recommend some other tool for this purpose?

[the remark this answers is not in the document]
Comment (mayaalsamarrai) [reply README.md]: I think for better readability we need to enclose captions and label names in quotes (at least when there is some other text near it), and use strictly the same names as on screenshots (e.g. "Payment fees (inc tax)").- this is not clear for me, please clarify

File: README.md [hunk 5]
```diff
@@ -1,34 +1,321 @@
...
 
-# Available resources
-* Module related service implementations as a <a href="https://www.nuget.org/packages/VirtoCommerce.OrderModule.Data" target="_blank">NuGet package</a>
-* API client as a <a href="https://www.nuget.org/packages/VirtoCommerce.OrderModule.Client" target="_blank">NuGet package</a>
-* API client documentation http://demo.virtocommerce.com/admin/docs/ui/index#!/Order_module
+The Order Management process in Vitro Commerce OMS is not coded and not pre-determined. This system is designed as an Order Details Editor with no validation logics available. The system is implied to be an additional storage for customer orders details.
+
+## Functional possibilities
+
+Vitro Commers Orders Module supports the following functionalities:
```
[the remark this answers is not in the document]
Comment (mayaalsamarrai) [reply README.md]: OK! Got it!!

File: README.md [hunk 6]
```diff
@@ -1,34 +1,321 @@
...
+
+1. Status update for each document type
+2. Document based order structure. The Order contains related documents such as Payments, Shipments, Addresses, etc.
+3. Ability to view and manage fulfillment, packages, pick-up and shipments documents
+4. Dynamic extensibility of the Order Documents (possibility to add random fields)
```
[the remark this answers is not in the document]
Comment (mayaalsamarrai) [reply README.md]: OK

File: README.md [hunk 7]
```diff
@@ -1,34 +1,321 @@
...
+1. Approved: Yes/No button
+1. Assigned to: displays the name of the assignee. The admin can select the assignee from the drop-down list
+1. Customer order Number: generated automatically by the system. The order number template can be specified by user
+    1. The user navigates to settings->Orders->General
+    1. The system opens the General Settings screen
+    1. The user specifies the Order number template
+
+![Fig.Customer Orders](/docs/media/screen-order-number-template.png)
+
+1. From – date of creation
```
[the remark this answers is not in the document]
Comment (mayaalsamarrai) [reply README.md]: It's fixed in the final version

File: README.md [hunk 8]
```diff
@@ -1,34 +1,321 @@
...
+1. Customer Name
+1. Discount
+1. Store
+
+### Widgets
+
+![Fig.Customer Orders](/docs/media/screen-order-widgets.png)
+
+1. Notification feed widget displays all notifications related to customer order
+1. Line Items widget display the list of selected items with details:
```
[the remark this answers is not in the document]
Comment (mayaalsamarrai) [reply README.md]: also corrected in the final version

File: README.md [hunk 9]
```diff
@@ -1,34 +1,321 @@
...
+
+    1. Total
+
+1. Changes widget displays the payment changes history(logs)
+1. Dynamic properties widget
+
+#### Order Shipment Document
+
+The user can view the Shipment document details by clicking on the ‘Shipment’ document
+The Order shipment document screen displays the following fields:
```
[the remark this answers is not in the document]
Comment (mayaalsamarrai) [reply README.md]: In the final version it's formulated in a different way, but any way the comment is not clear for me

File: README.md [hunk 10]
```diff
@@ -1,34 +1,321 @@
...
+
+    1. Discount total
+
+    1. Tax total
+
+    1. Total
+
+## Roles and Permissions
+
+In order to manage Virto Commerce functionality each user must be assigned to at least one role. Each role provides the user with certain access permissions. These permissions allow or restrict the user's access to functionalities within the Virto Commerce client application.
```
[the remark this answers is not in the document]
Comment (mayaalsamarrai) [reply README.md]: Do I have to add the "server" ?

File: README.md (hunk 1 above)
Comment (mayaalsamarrai) [reply README.md]: OK, got it

File: README.md (hunk 2 above)
Comment (mayaalsamarrai) [reply README.md]: Database Model and it was added by Maxim. I don't understand what you exactly mean

File: README.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (yecli) [reply README.md]: We could remove "client"

**Comment by yecli** [comment]
Closed as it is duplicate of #100

**Pull request closed without merging**